index.html

```diff
@@ -5425,7 +5425,7 @@ <h5>Conclusion</h5>
 </li>
 <li><a data-ucr-role="tag" href="">accessibility-potential</a>
 Current implementations can only represent changes in the map state in a semantically meaningful form
-to a limited degree via such mechanisms as [[[ARIA]]] live regions.
+to a limited degree via such mechanisms as [[WAI-ARIA]] <a href="https://www.w3.org/TR/wai-aria/#dfn-live-region">live regions</a>.
```

Comment:
```suggestion
to a limited degree via such mechanisms as [[[wai-aria-1.1]]] <a href="https://www.w3.org/TR/wai-aria/#dfn-live-region">live regions</a>.
```

(The W3 URL without the version, in contrast, always links to the latest stable version! But SpecRef doesn't seem to have a way to do that.)